crossReference.conf

```diff
@@ -0,0 +1,55 @@
+
+  { source: "Sample Models/Biology/Evolution/Unverified/Divide The Cake.nlogo",
+     referenceIn: "Curricular Models/EACH/Unverified" }
+
+  { source: "Sample Models/Networks/Team Assembly.nlogo",      referenceIn: "Sample Models/Social Science" }
+
+  { source: "Code Examples/Extensions Examples/vid/Video Camera Example.nlogo", referenceIn: "Code Examples/Extension Examples/bitmap" }
+
+  // BEAGLE curricular models
+  { source: "Sample Models/Biology/Wolf Sheep Predation.nlogo",                        referenceIn: "Curricular Models/BEAGLE Evolution" }
+  { source: "Sample Models/Biology/Evolution/Genetic Drift/GenDrift T interact.nlogo", referenceIn: "Curricular Models/BEAGLE Evolution" }
+  { source: "Sample Models/Biology/Evolution/Bug Hunt Speeds.nlogo",                   referenceIn: "Curricular Models/BEAGLE Evolution" }
+  { source: "Sample Models/Biology/Evolution/Bug Hunt Camouflage.nlogo",               referenceIn: "Curricular Models/BEAGLE Evolution" }
+  { source: "HubNet Activities/Unverified/Guppy Spots HubNet.nlogo",                   referenceIn: "Curricular Models/BEAGLE Evolution" }
+  { source: "Sample Models/Biology/Daisyworld.nlogo",                                  referenceIn: "Curricular Models/BEAGLE Evolution" }
+  { source: "Sample Models/Biology/Evolution/Mimicry.nlogo",                           referenceIn: "Curricular Models/BEAGLE Evolution" }
+  { source: "Sample Models/Biology/Evolution/Altruism.nlogo",                          referenceIn: "Curricular Models/BEAGLE Evolution" }
+  { source: "Sample Models/Biology/Evolution/Cooperation.nlogo",                       referenceIn: "Curricular Models/BEAGLE Evolution" }
+  { source: "HubNet Activities/Bug Hunters Camouflage HubNet.nlogo", referenceIn: "Curricular Models/BEAGLE Evolution/HubNet Activities" }
+
+  // copy BEAGLE HubNet models to hubnet activities
+  { source: "Curricular Models/BEAGLE Evolution/HubNet Activities/Bird Breeders HubNet.nlogo",           referenceIn: "HubNet Activities" }
+  { source: "Curricular Models/BEAGLE Evolution/HubNet Activities/Bug Hunters Competition HubNet.nlogo", referenceIn: "HubNet Activities" }
+  { source: "Curricular Models/BEAGLE Evolution/HubNet Activities/Critter Designers HubNet.nlogo",       referenceIn: "HubNet Activities" }
+  { source: "Curricular Models/BEAGLE Evolution/HubNet Activities/Fish Spotters HubNet.nlogo",           referenceIn: "HubNet Activities" }
+
+  // Oil Cartel HubNet to HubNet Activities
+  { source: "Sample Models/Social Science/Oil Cartel HubNet.nlogo", referenceIn: "HubNet Activities" }
+
+  // IABM Textbook models duplicated in Sample Models and Code Examples
+  { source: "IABM Textbook/chapter 3/El Farol Extensions/El Farol.nlogo",  referenceIn: "Sample Models/Social Science" }
+  { source: "IABM Textbook/chapter 3/DLA extensions/DLA Simple.nlogo",     referenceIn: "Sample Models/Chemistry & Physics/Diffusion Limited Aggregation" }
```

Comment:
This line is producing an error. Should it be (capitalize "extensions"):

`{ source: "IABM Textbook/chapter 3/DLA Extensions/DLA Simple.nlogo",     referenceIn: "Sample Models/Chemistry & Physics/Diffusion Limited Aggregation" }`

I assume that means this new method is case-sensitive while the old was not?